### ISSUE TITLE
Online Mode Supported?

### ISSUE THREAD

**Issue by SugarD-x** [Tnze/go-mc#26]
Is online mode supported out of the box, or does it have to be manually added? I was able to get offline mode working just fine, but online mode keeps throwing me authentication errors using a legitimate Minecraft account. Specifically:
`bot: encryption fail: login fail: auth fail: {"error":"IllegalArgumentException","errorMessage":"Access Token can not be null or empty."}
exit status 1`

I've gone through and set up the account information everywhere I can find, but I'm wondering if I could be missing a file or setting somewhere. I'm still a bit new to this, so I apologize if this is a stupid question. For information, this is for connecting to a vanilla 1.14.4 server on localhost with online-mode set to true.

Edit: On the server's end, it doesn't show me connecting, but it does show me disconnecting. No errors come up there.

**Comment by Tnze** [comment]
Are you code with `bot` package or just running a example in `cmd`?
There is an example of online mode at https://github.com/Tnze/go-mc/blob/7d31acbb483689e47aeb0251c2a1d08b2f2c03ca/bot/example_test.go#L47

Make sure you have set `c.AsTk`.

**Comment by SugarD-x** [comment]
I'm running the daze example from cmd. That and luncher were the only two that would run for me without throwing an error about not being a main package. Is there a way to specifically run the example without creating a new package?

My apologies, I'm brand new to go-lang, and I'm still trying to wrap my head around how this project works. I'm attempting to get the basics down and work out a simple online mode connection before making anything with it so I don't create a bigger mess.

**Comment by Tnze** [comment]
```go
c := NewClient()

	//Login Mojang account to get AccessToken
	auth, err := yggdrasil.Authenticate("Your E-mail", "Your Password")
	if err != nil {
		panic(err)
	}

	c.Auth.UUID, c.Name = auth.SelectedProfile()
	c.AsTk = auth.AccessToken()

	//Connect server
	err = c.JoinServer("localhost", 25565)
	if err != nil {
		log.Fatal(err)
	}
```
To set c.AsTk, write `c.AsTk = auth.AccessToken()` into the daze.go

You get accessToken by yggdrasil.Authenticate() as an easy way.

**Comment by Tnze** [comment]
The example can't directly run.

**Comment by SugarD-x** [comment]
Thank you!!! Apparently I was on the right track and didn't realize it. I had tried messing with `daze.go` previously to do something similar to your example, but I had trouble getting it to work. What I didn't realize until now is that I needed to import `github.com/Tnze/go-mc/yggdrasil` and change the server login section's `err` name so it didn't match the one used in the access token login, else it would throw errors when running. You are awesome!!! Thank you again!!!
